### PR TITLE
[4.0] Draggable

### PR DESCRIPTION
**Pull request by brianteeman**
The following admin views have not been converted to use draggable for sorting so the sorting is currently broken

```
index.php?option=com_languages&view=languages
index.php?option=com_users&view=levels
index.php?option=com_content&view=featured
```

Once applied you can sort the rows in these views and the sort order will be saved.

Pull Request for Issue #21787

### CODE REVIEW

**Comment by bees4ever** [comment]
Unfortunately `index.php?option=com_content&view=articles` and `index.php?option=com_content&view=featured` is not draggable but `index.php?option=com_languages&view=languages` and `index.php?option=com_users&view=levels` are draggable.
 
draggable_failed2

**Comment by bees4ever** [comment]
I have tested this item :red_circle: unsuccessfully on 37b176159a5997182006ed11416fa6010385a5a7<br /><br />See https://issues.joomla.org/tracker/joomla-cms/22049#event-390511<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22049">issues.joomla.org/tracker/joomla-cms/22049</a>.</sub>

**Comment by ThomasFried** [comment]
I have tested this item :red_circle: unsuccessfully on 37b176159a5997182006ed11416fa6010385a5a7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22049">issues.joomla.org/tracker/joomla-cms/22049</a>.</sub>

**Comment by ufuk-avcu** [comment]
I have tested this item :red_circle: unsuccessfully on 37b176159a5997182006ed11416fa6010385a5a7<br /><br />This view is not draggable: index.php?option=com_content&view=featured

The other both views are draggable<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22049">issues.joomla.org/tracker/joomla-cms/22049</a>.</sub>

**Comment by brianteeman** [comment]
@bees4ever I can see from your movie that you forgot to click on the column header - this is required before sorting is possible. Basically you can only sort a column by ordering (drag'n'drop) if you have already sorted the column by ordering. 
@ufuk-avcu @ThomasFried  Can you test again and see if you made the same error

**Comment by brianteeman** [comment]
can this please be retested. i am convinced it works and that the testers did not sort the column first.

**Comment by bees4ever** [comment]
I have tested this item :white_check_mark: successfully on 27be54546a23846b3415e762f08c882b983fcf6f<br /><br />With the hint from @brianteeman both views (`index.php?option=com_content&view=articles` and `index.php?option=com_content&view=featured`) worked for me. 

Thank you<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22049">issues.joomla.org/tracker/joomla-cms/22049</a>.</sub>

File: administrator/components/com_content/tmpl/featured/default.php [hunk 1]
```diff
@@ -131,7 +132,7 @@
 								</th>
 							</tr>
 						</thead>
-						<tbody>
+						<tbody <?php if ($saveOrder) :?> class="js-draggable" data-url="<?php echo $saveOrderingUrl; ?>" data-direction="<?php echo strtolower($listDirn); ?>"<?php endif; ?>>
```
Comment (Quy) [inline administrator/components/com_content/tmpl/featured/default.php]: Add space after `:` and remove space before `class` to change it like this: `<?php if ($saveOrder) : ?>class="js-draggable"`

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**